### DISCUSSION TITLE
Adobe Animate CC EXE Crashes immediatelly

### DISCUSSION THREAD

**Discussion by kutluhanaykut** [airsdk/Adobe-Runtime-Support#2958] [General]
Hi,

Application runs inside Animate with no error. I am using some TXT files to read and wrtie and load some external image and mp3.
When  i compile windows projector. Projector closes immediately. with no error...

TIP: I managed to create a working exe by building and .air file (and/or windows air installer) and installing it to system with air standalone...   But it is not a perfect solution for this issue. 

What could be happening or not happening

**Comment by balaedit** [comment]
Can you share the sample code using. I found no issue in loading text file, images or mp3/videos.

**Reply by kutluhanaykut** [reply]
[cark v3.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/13610662/cark.v3.zip)

It can work fine inside animate as i said.. no problem in loaading or even saving file to desktop

**Reply by Jason0247** [reply]
The uploaded project is not a complete one

**Reply by balaedit** [reply]
I had tested the app. It is working properly on testing. I had swapped your mp3,jpg and other to my assets. It was working not a single time it crashed.

**Reply by kutluhanaykut** [reply]
Strange it is still not working.. her i guess because of my system or the compiler SDK.. :(

I managed to make it work by installing air runtime and installing the app with this. Afterwards it was working.. just on my system.. 

When i transferred to use it on another computer i had to install air runtime first  then it was working
